embedx/config.schema.json

```diff
@@ -1350,6 +1350,16 @@
         "admin": {
           "type": "object",
           "properties": {
+            "access_log": {
```

Comment:
```suggestion
            "request_log": {
```
I think request log makes more sense than access log.

Reply:
And it would be good to add a description here.

Reply:
Fixed in last commit.